apps/graphql/src/definitions/Build.js

```diff
@@ -37,13 +52,20 @@ export const typeDefs = gql`
     status: BuildStatus!
     "Build name"
     name: String!
+    "Build stats"
+    stats: BuildStats!
   }
 
   type BuildResult {
     pageInfo: PageInfo!
     edges: [Build!]!
   }
 
+  extend type Query {
+    "Get a build"
+    build(ownerLogin: String!, repositoryName: String!, number: Int!): Build
```

Comment:
I don't get why we don't use the repository to get the build. Logic is duplicated.